volume

```diff
@@ -601,6 +603,9 @@ while getopts ":ac:d:e:hi:lmno:ps:S:t:u:v:x:X:y" o; do
         m)
             opt_mute_volume=true
             ;;
+        M)
+            mixer="${OPTARG}"
```

Comment:
Consider using `mixer="${OPTARG@Q}"`, this will prevent mixer controls that contain spaces from not working